common/src/main/java/com/revenuecat/purchases/common/logUtils.kt

```diff
@@ -9,8 +9,12 @@ fun debugLog(message: String) {
     }
 }
 
-fun log(message: String) {
-    Log.w("[Purchases] - INFO", message)
+fun infoLog(message: String) {
+    Log.i("[Purchases] - INFO", message)
+}
+
+fun warnLog(message: String) {
+    Log.w("[Purchases] - WARNING", message)
```

Comment:
following purchases-ios Logging Part 1, should the prefix be "WARN" or "WARNING"?

Reply:
Keeping it consistent I'll change to `WARN`